python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py

```diff
@@ -95,6 +95,8 @@ def setup_ert(tmp_path_factory):
                 {ConfigKeys.NAME: "poly_eval", ConfigKeys.PATH: "POLY_EVAL"}
             ],
             ConfigKeys.SIMULATION_JOB: [{ConfigKeys.NAME: "poly_eval"}],
+            ConfigKeys.HOOK_WORKFLOW_KEY: ["obs_scaling", "POST_SIMULATION"],
```

Comment:
Could you explain why the tests are changed?

Reply:
Of definitively! Just was a bit curious as by the title of the PR I would not expect these kind of changes to the tests.